packages/fx-core/src/plugins/solution/fx-solution/arm.ts

```diff
@@ -306,13 +301,13 @@ export async function copyParameterJson(
   targetEnvName: string,
   sourceEnvName: string
 ) {
-  if (!isMultiEnvEnabled() || !targetEnvName || !sourceEnvName) {
+  if (!targetEnvName || !sourceEnvName) {
```

Comment:
I'm not sure why there's a `isMultiEnvEnabled()` check previously, is there any side effect of removing this check?

Reply:
this function trigger on create a new ENV, and will copy azure.parameter.dev.json to a new env json file